src/Kafka/CreateKafkaInstanceWithSizes/components/FieldCloudRegion.tsx

```diff
@@ -0,0 +1,86 @@
+import { FormGroup, HelperText, HelperTextItem } from "@patternfly/react-core";
+import { VoidFunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+import { CloudRegionProps, CloudRegionSelect } from "./CloudRegionsSelect";
+
+export type FieldCloudRegionProps = {
+  validity: "valid" | "region-unavailable" | "required";
+} & Omit<CloudRegionProps, "validated" | "placeholderText">;
+export const FieldCloudRegion: VoidFunctionComponent<FieldCloudRegionProps> = ({
+  value,
+  regions,
+  onChange,
+  isDisabled,
+  validity,
+}) => {
+  const { t } = useTranslation("create-kafka-instance-with-sizes");
+
+  const allRegionsUnavailable = regions
+    ? regions.every(({ isDisabled }) => isDisabled === true) ||
+      regions.length === 0
+    : true;
+
+  const disableControl =
+    isDisabled || allRegionsUnavailable || regions?.length === 0;
+
+  const someRegionsUnavailable =
+    !allRegionsUnavailable &&
+    regions?.some(({ isDisabled }) => isDisabled === true);
+
+  const validation =
+    validity !== "valid"
+      ? value !== undefined
+        ? "error"
+        : "default"
+      : "default";
+
+  const placeholder =
+    allRegionsUnavailable && regions
+      ? t("regions_temporarily_unavailable")
+      : t("select_region");
+
+  const helperTextInvalid = allRegionsUnavailable
+    ? t("all_region_unavailable_helper_text")
+    : t("common:required");
+
+  return (
+    <FormGroup
+      data-testid="cloudRegion"
+      label={t("cloud_region")}
+      fieldId="form-cloud-region-option"
+      isRequired
+      validated={validation}
+      helperText={
+        regions && (allRegionsUnavailable || someRegionsUnavailable) ? (
+          <HelperText className={"pf-c-form__helper-text"}>
+            <HelperTextItem variant="warning" hasIcon>
+              {someRegionsUnavailable
+                ? t("some_region_unavailable_helper_text")
+                : t("all_region_unavailable_helper_text")}
```

Comment:
When I look at [No regions returned from API for a provider - Standard](https://619e1921b3e38b003afeb2be-tvvapjvikl.chromatic.com/?path=/story/kafka-create-kafka-instance-with-sizes-stories-form-load--no-regions-returned-from-api-for-a-provider-on-form-load), I see this helper text display with the warning variant. 

But I expect this helper text to display using the error variant. Only the  `some_region_unavailable_helper_text` helper text should use the warning variant.

<img width="558" alt="image" src="https://user-images.githubusercontent.com/21063328/166305905-99cacc25-c171-4f38-8fdd-3a5adadb2f04.png">